src/main/resources/org/broadinstitute/barclay/helpTemplates/generic.template.html

```diff
@@ -31,7 +31,7 @@
 	<#macro argumentDetails arg>
 		<hr style="border-bottom: dotted 1px #C0C0C0;" />
 		<h3><a name="${arg.name}">${arg.name} </a>
-			<#if arg.synonyms??> / <small>${arg.synonyms}</small></#if>
+			<#if arg.synonyms??><#if arg.synonyms != "NA"> / <small>${arg.synonyms}</small></#if></#if>
```

Comment:
Since we're changing this line anyway, can you remove the outer "#<if arg.synonyms??>". I think it probably is and always was unnecessary.